static/templates/left_sidebar.hbs

```diff
@@ -69,6 +55,31 @@
                 <span class="arrow drafts-sidebar-menu-icon"><i class="zulip-icon zulip-icon-ellipsis-v-solid" aria-hidden="true"></i></span>
             </li>
         </ul>
+
+    <div id="pm_stream_scroll_container" class="scrolling_list" data-simplebar>
+        <div id="private_messages_section" class="zoom-out hidden-for-spectators">
+            <div id="private_messages_list">
+                <div id="private_messages_section_header" class="zoom-out zoom-in-sticky">
+                    <div class="left_part">
+                        <a class="hide_more_private_messages zoom-out-hide">
+                            <i class="fa fa-chevron-left" aria-hidden="true"></i>
+                            {{t 'BACK' }}
+                        </a>
+                        <i id="toggle_private_messages_section_icon" class="fa fa-lg fa-caret-down toggle_private_messages_section zoom-in-hide" aria-hidden="true"></i>
+                        <span id="more_private_messages_sidebar_title" class="zoom-out-hide">{{t 'Private messages' }}</span>
+                        <h4 class="sidebar-title toggle_private_messages_section zoom-in-hide" data-tippy-content="{{t 'Private messages' }}">{{t 'PRIVATE MESSAGES' }}</h4>
+                    </div>
+                    <div class="right_part">
+                        <span class="unread_count"></span>
+                        <a href="#narrow/is/private" id="show_all_private_messages">
+                            <i class="fa fa-align-right" aria-hidden="true"></i>
+                        </a>
+                    </div>
+                </div>
+                <div id="private_messages_default"></div>
```

Comment:
I felt like the old name for this was weird. Should we change it to `private_messages_list`?

Reply:
Yeah that sounds better.

Reply:
Cool, I went ahead and renamed this to `private_messages_list`; Thanks Aman, Tim.